docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md

```diff
@@ -49,19 +45,14 @@ Create a log dir for the supervisor logs:
 mkdir /home/git/gitea/log/supervisor
 ```
 
-Open supervisor config file in a file editor:
-```
-sudo vim /etc/supervisor/supervisord.conf
-```
-
 Append the configuration from the sample
-[supervisord config](https://github.com/go-gitea/gitea/blob/master/contrib/supervisor/gitea).
+[supervisord config](https://github.com/go-gitea/gitea/blob/master/contrib/supervisor/gitea) to `/etc/supervisor/supervisord.conf`.
 
-Change the user (git) and home (/home/git) settings to match the deployment
-environment. Change the PORT or remove the -p flag if default port is used.
+Using your favorite editor (such as `vim`), change the user (git) and home
+(/home/git) settings to match the deployment environment. Change the PORT
+or remove the -p flag if default port is used.
 
 Lastly enable and start supervisor at boot:
 ```
-sudo systemctl enable supervisor
-sudo systemctl start supervisor
+sudo systemctl enable supervisor --now
```

Comment:
same as above

Reply:
@6543 basically `systemctl enable --now` enables the service and immediately starts it